client/payment-methods-map.tsx

```diff
@@ -156,18 +166,20 @@ const PaymentMethodInformationObject: Record<
 		currencies: [ 'USD', 'CAD' ],
 		stripe_key: 'affirm_payments',
 		allows_manual_capture: false,
+		allows_pay_later: true,
 	},
 	afterpay_clearpay: {
 		id: 'afterpay_clearpay',
-		label: __( 'Afterpay/Clearpay', 'woocommerce-payments' ),
+		label: __( 'Afterpay / Clearpay', 'woocommerce-payments' ),
```

Comment:
Also, are there opportunities for dynamic branding here?

Reply:
Replacing Afterpay / Clearpay depending on the country is also required on other PR's in the project, wherever we use the brand name. I felt that we may be duplicating it if I do it individually in this PR. Hence I checked with @mgascam, and it was [decided we will create a separate ticket ](p1685434656041599/1685431915.076569-slack-C02C00HCJ8J)for this, probably handled after MVP.

Reply:
A quick note that the icon used for Afterpay/Clearpay currently only has Afterpay in it. So technically, we should also have 2 icons that are used dynamically based on same rules.

Reply:
That works for me, thanks!

Reply:
I have created an [issue here](https://github.com/Automattic/woocommerce-payments/issues/6425) for the Afterpay / Clearpay switching.